### PR TITLE
[DNM] Add: Admin Call Mob To Coordinate Verb

### PR DESCRIPTION
**Pull request by Cupax3**
For events and stuff, not refined, no warranty etc etc

## About The Pull Request
Adds a verb that allows admins through a very clunky interface to send mobs to attack people

## Why It's Good For The Game
Attacking people with mobs is FUN

## Changelog
:cl:
add: "Call Mob to Coordinate" Admin Event Verb
/:cl:

### Currently only intended for events via testmerge

### CODE REVIEW

File: code/modules/admin/admin_verbs.dm [hunk 1]
```diff
@@ -83,7 +83,9 @@ GLOBAL_PROTECT(admin_verbs_admin)
 	/client/proc/SpawnAbno,
 	/client/proc/ClearAbno,
 	/client/proc/FullUnderstandAbno,
-	/client/proc/BreachAbno
+	/client/proc/BreachAbno,
+	// Event stuff
+	/client/proc/admin_ai_move_to_coordinate
```
Comment (Gboster-0) [inline code/modules/admin/admin_verbs.dm:88]:
```suggestion
	/client/proc/admin_ai_move_to_coordinate,
```

Comment (Cupax3) [reply code/modules/admin/admin_verbs.dm:88]: :+1:

**Comment by Cupax3** [comment]
This is currently semi-weird behaviour: If the mobs cannot find a path to the coordinate, you get no indication /and/ the mobs do not even attempt to reach it, but that is a deeper AI issue.

**Comment by Cupax3** [comment]
Continuing: Sandbags should not interfere with pathfinding somewhat.

**Comment by Cupax3** [comment]
thanks GitHub

**Pull request closed without merging**